BEIMA.Backend/MongoService/MongoDefinition.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace BEIMA.Backend.MongoService
+{
+    /// <summary>
+    /// Helps to store the instance of the Mongo client being used.
+    /// </summary>
+    /// <remarks>
+    /// The purpose of this class is to allow us to switch out the default
+    /// Mongo connector with a mock Mongo connector for testing.
+    /// </remarks>
+    public static class MongoDefinition
+    {
+        /// <summary>
+        /// Gets and sets the currently defined instance of the Mongo connector
+        /// </summary>
+        public static IMongoConnector MongoInstance { get; set; } = MongoConnector.Instance;
```

Comment:
Thought this was the best way to do this, but I wanted to be able to switch out the running instance of the MongoConnector with a Mock instance. So the endpoint will call this instead of MongoConnector.Instance, and that way it will use whichever instance is set here. By default it is set to the actual mongo instance.